README.md

```diff
@@ -1,150 +1,235 @@
-3. Angular (https://www.npmjs.com/package/angular) - for building and running our web application
-4. Angular CLI (https://cli.angular.io/)
-4. Git CLI (https://git-scm.com/downloads) - for downloading the source files for this tutorial
-5. A Java IDE (Eclipse, Visual Studio Code, Netbeans etc) - for coding our tutorial
-6. Connection to the Internet - to pull down the required dependancies
-
-
-### To Note!
-
-We understand that normally you would have a separate repository for each microservice but to make this tutorial easier to consume we have decided to keep all the source in one repository to save the user having to download multiple repositories.
-
-We are planning to extent this tutorial to use Docker, Kuberneties and other technologies but we wanted to limit the prerequisites so that users can get up and ready quickly.
-
-## Information About The Files We Have Provided
-We have provided some file for you to make this tutorial easier to consume that I will talk about here.
-Firstly we have provided you with the basic directory structure for all the microservices.
-Then we have provided you with a pom.xml file that is used by maven to build your microservices and download all the required dependancies. In this file you can set what the application is named `<app.name>LibertyMicroServiceOne-1.0</app.name>`, the ports assigned to Liberty 
-```
-<testServerHttpPort>9090</testServerHttpPort> <testServerHttpsPort>9444</testServerHttpsPort>
-```
-the dependancies you require like microProfile 1.2
-```
-<dependency>
-    <groupId>org.eclipse.microprofile</groupId>
-    <artifactId>microprofile</artifactId>
-    <version>1.2</version>
-    <scope>provided</scope>
-    <type>pom</type>
-</dependency>
-```
-and the applicaiton server you are using so in our case open Liberty
+###About this module
+
+This module is a continuation of module 3 however, the finished module 3 code has been provided in this module 4 branch.
+
+This module will demonstrate MicroProfile Config. MP Config is a solution to externalise configuration from microservices, enabling users to separate configuration from code to enable microservices to be successfully run in different environments requiring different configurations. This helps to ensure the high portability of microservices.
+
+1.  Firstly, the static configuration properties in a static configuration properties file need to be set (this could be one of three pre-existing config files).
+2.  Following this, a custom configuration file needs to be created to enable dynamic configuration properties (Properties that can be updated without having to restart the server).
+3.  Finally, the new dynamic and static configuration properties need to be tested.
+
+If more help is needed in regards to any of the concepts and methods used in this module, please see the two interactive guides on http://OpenLiberty.io.guides : 
+1.  Using MicroProfile Config for static configuration injection [LINK]
```

Comment:
There is no links here but you have specified [LINK]

Reply:
This LINK is where the url link to the open liberty guides will be placed. However they have not been published as of yet and so I could not include the link here. I will replace this once these guides have been published with the correct URL.